flake.nix

```diff
@@ -134,7 +134,8 @@
             nativeBuildInputs = with pkgs; [ pkg-config ];
             CARGO_BUILD_TARGET = targetConfig.rustTarget;
             CARGO_BUILD_RUSTFLAGS = [ "-Ctarget-feature=+crt-static" ];
-            GIT_REV = self.rev or self.dirtyRev or "dirty";
+            GIT_REV = self.rev or self.dirtyRev or "unknown-rev";
```

Comment:
Ah this change isn't necessary, only the addition below -- I just had a different name and during the merge conflict I kept the wrong one.